server/routes/list.js

```diff
@@ -22,6 +22,7 @@ router.delete('/:id/delete', authorize, deleteListById);
 router.patch('/:id/update', authorize, updateListById);
 router.get('/:id/data', authorize, getListData);
 router.patch('/:id/update-item', authorize, updateListItem);
-router.patch('/:id/vote-for-item', authorize, voteForItem);
+router.patch('/:id/set-vote', authorize, voteForItem);
+router.patch('/:id/clear-vote', authorize, voteForItem);
```

Comment:
Hmm, I know we discussed about it but it looks strange: we have two routes and one controller which check path (req.route.path.includes('set-vote')) and run appropriate logic. Maybe it would be better to use separate controllers? We have such cases: fetchListsMetaData and fetchArchiveListMetaData... and the same for cohort...